build/build.xml

```diff
@@ -1161,6 +1155,15 @@
       </condition>
     </fail>
 
+    <copy todir="windows/work">
+      <fileset dir="windows/dist" includes="*.dll" />
+    </copy>
```

Comment:
If these files are in the bin directory, are they still needed here? I haven't checked, but won't they be shipped twice in the resulting distribution? Or are they just duplicated in the build dir?

Reply:
Yes, they will be shipped twice.

It's quite weird how Windows and `java` decides how to look for native libraries... in theory what you say is correct but I'm quite afraid of removing the two dlls from where they stayed for many years... so in the end I opted to keep in both places and consume a few kilobytes on the distribution files.